### ISSUE TITLE
plot.topo() problem with non-increasing x or y value

### ISSUE THREAD

**Issue by dankelley** [dankelley/oce#1074]
I'm not sure if this is a general problem or just something with this downloaded file, but since I will soon be suggesting that people employ `download.topo`, this has to be sorted out.
```R
> library(oce)
Loading required package: gsw
Loading required package: testthat
> topoFile<-download.topo(-180,180,-90,90,12,"~/data/topo/")
> read.topo(topoFile)
topo object, from file '~/data/topo//topo_180W_180E_90S_90N_12min_gmt.nc', has data as follows.
   longitude: -180.0, -179.8, ..., 179.8, 180.0 (length 1801)
   latitude: -90.0, -89.8, ..., 89.8, 90.0 (length 901)
   z, a 1801x901 array with value 2744 at [1,1] position
Warning messages:
1: In read.topo(topoFile) :
  offsetting longitude of a netcdf topo file by half a step
2: In read.topo(topoFile) :
  offsetting latitude of a netcdf topo file by half a step
> topo <- read.topo(topoFile)
Warning messages:
1: In read.topo(topoFile) :
  offsetting longitude of a netcdf topo file by half a step
2: In read.topo(topoFile) :
  offsetting latitude of a netcdf topo file by half a step
> plot(topo)
Error in contour.default(xx, yy, zz, levels = 0, drawlabels = FALSE, add = TRUE,  : 
  increasing 'x' and 'y' values expected
> 
```

**Comment by dankelley** [comment]
Note: `imagep(topo)` works fine.

**Comment by dankelley** [comment]
Fixed in `develop` commit 7b5ef1f20a796e79e0032e612faf600ef0eda5cc. The problem was because I was repeating the world, to let the view go from -180/180 to 0/360.
